docs/content/3.options/1.vue-i18n.md

```diff
@@ -6,26 +6,37 @@ Related Vue I18n options.
 
 ## `vueI18n`
 
-- type: `object` or `string`
-- default: `{}`
-
-Configuration for the `vue-i18n` library that is used internally by this module. See full documentation at https://vue-i18n.intlify.dev/api/general.html#createi18n
-
-::alert{type="info"}
-
-It's also supported to set this property to a path to a local configuration file. The file needs to export a function or plain object. If a function, it will be passed a Nuxt Context as a parameter. It's necessary to use that approach when overriding more complex types (like functions) that can't be stringified correctly.
-
-```ts {}[~/vue-i18n.options.ts]
-import type { I18nOptions } from 'vue-i18n'
-import type { NuxtApp } from 'nuxt/dist/app/index'
+- type: `object`
+- default: `undefined`
+
+Build-time configuration for the `vue-i18n` options that is used internally by this module. See full documentation at https://vue-i18n.intlify.dev/api/general.html#createi18n
+
+Configuration for `createI18n` can be passed using a configuration file. By default, the module will scan for a `i18n.config.{js,mjs,ts}` if nothing is specified.
+
+```ts {}[nuxt.config.ts]
+export default defineNuxtConfig({
+  modules: [
+    '@nuxtjs/i18n'
+  ],
+  i18n: {
+    vueI18n: { configPath: './nuxt-i18n.js' } // custom path example
```

Comment:
The `vueI18n` option may no longer be optional. It's recommended to autoload `i18n.config.ts` if we have a config that conforms to the convention. If we have a nonconventional config file name, we can specify it in the `vueI18n` option!

Reply:
ah, sorry, Never mind this comment.
I was misleading. 😅